src/commands/mod.rs

```diff
@@ -1143,6 +1158,46 @@ fn cmd_config_list(
     Ok(())
 }
 
+fn cmd_config_get(
+    ui: &mut Ui,
+    command: &CommandHelper,
+    args: &ConfigGetArgs,
+) -> Result<(), CommandError> {
+    let value = command
+        .settings()
+        .config()
+        .get_string(&args.name)
+        .map_err(|err| match err {
+            config::ConfigError::Type {
+                origin,
+                unexpected,
+                expected,
+                key,
+            } => {
+                let expected = format!("a value convertible to {expected}");
+                // Copied from `impl fmt::Display for ConfigError`. We can't use
+                // the `Display` impl directly because `expected` is required to
+                // be a `'static str`.
+                let mut buf = String::new();
+                use std::fmt::Write;
+                write!(buf, "invalid type: {unexpected}, expected {expected}").unwrap();
+                if let Some(key) = key {
+                    write!(buf, " for key `{key}`").unwrap();
+                }
+                if let Some(origin) = origin {
+                    write!(buf, " in {origin}").unwrap();
+                }
+                // Ensure that we return a `config::ConfigError` so that this
+                // error message is later affixed with the link to the jj config
+                // documentation.
+                config::ConfigError::Message(buf.to_string())
```

Comment:
nit: `CommandError::ConfigError()` works, but I think using `config::ConfigError` is also good.